### PR TITLE
feat: started a graphql fixture library, so we dont repeat ourself

### PR DESCRIPTION
**Pull request by Itrulia**

### CODE REVIEW

File: libs/testing/fixtures/graphql/src/index.ts [hunk 1]
```diff
@@ -6,3 +6,15 @@ export * from './lib/article/article'
 export * from './lib/blocks/blocks'
 export * from './lib/page/page'
 export * from './lib/peer/peer'
+export * from './lib/text/text'
+export * from './lib/challenge/challenge'
+
+// comments
+export * from './lib/comment/anonymousComment'
```
Comment (Itrulia) [inline libs/testing/fixtures/graphql/src/index.ts:13]: file naming should not be camel case

File: libs/testing/fixtures/graphql/src/lib/comment/verifiedUserComment.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import {CommentListQuery} from '@wepublish/website/api'
+import {image} from '../image/image'
+import {text} from '../text/text'
+
+export const verifiedUserComment: CommentListQuery['comments'][number] = {
+  id: 'verified',
+  parentID: 'cljgx3n3i382572shctpgd5gg0',
+  peerId: null,
+  overriddenRatings: [],
+  user: {
```
Comment (Itrulia) [inline libs/testing/fixtures/graphql/src/lib/comment/verifiedUserComment.ts:10]: Should probably be split into seperate mock

File: libs/testing/fixtures/graphql/src/lib/blocks/blocks.ts [hunk 3]
```diff
@@ -0,0 +1,1099 @@
+import {image} from '../image/image'
+import {poll} from '../poll/poll'
+import {event} from '../event/event'
+import {Block, Exact} from '@wepublish/website/api'
+
+export const blocks: Exact<Block>[] = [
+  {
```
Comment (Itrulia) [inline libs/testing/fixtures/graphql/src/lib/blocks/blocks.ts:7]: should probably be splitoff into multiple mocks

File: libs/testing/fixtures/graphql/src/lib/comment/verifiedUserComment.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import {CommentListQuery} from '@wepublish/website/api'
+import {image} from '../image/image'
+import {text} from '../text/text'
+
+export const verifiedUserComment: CommentListQuery['comments'][number] = {
```
Comment (Itrulia) [inline libs/testing/fixtures/graphql/src/lib/comment/verifiedUserComment.ts:5]: Typing should probably be `Exact<FullCommentFragment>`

**Comment by elias-summermatter** [comment]
@Itrulia What is the state here?

**Comment by Itrulia** [comment]
@elias-summermatter still WIP

**Pull request closed without merging**